docs/data-sources/compute_instances.md

```diff
@@ -0,0 +1,112 @@
+---
+subcategory: "Elastic Cloud Server (ECS)"
+---
+
+# flexibleengine\_compute\_instances
```

Comment:
- **\\** is unnecessary
- should we add "_v2" suffix to keep consistent with other resources and data sources?

Reply:
"\" is removed.
In my opinion Terraform abstract the API call so a resource should not contain the version of the API Terraform is calling. And a change in the API should result in a new version of the terraform provider so user can select the appropriate version for their usage. What do you think about it ?

Reply:
yes, we can add new resource or data source without API version.